modules/redis/variables.tf

```diff
@@ -83,3 +83,19 @@ variable "redis_configs" {
     maxmemory-policy = "allkeys-lfu"
   }
 }
+
+variable "enable_replicas" {
+  description = "Enable read replicas"
+  type        = bool
+  default     = false
+}
+
+variable "replica_count" {
+  description = "The number [1-5] of replica nodes. Defaults to 2."
+  type        = number
+  default     = 2
```

Comment:
I think this will fail if you disable read replicas. From the doc (https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/redis_instance#replica_count): "If read replicas are not enabled for a Standard Tier instance, the only valid value is 1 and the default is 1".
Since there is a strong dependency between these two, cloud it be better to combine them into one variable?

Reply:
Nice catch! I evidently read that doc a bit too quickly...
`replica_count = 0` in order to disable seems a bit weird, so I propose we keep the terraform API.

Now it defaults to 1 instead of 2.

Thoughts?

Reply:
Nice 👍